back/database/seeds/MessagesTableSeeder.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use Illuminate\Database\Seeder;
+use Carbon\Carbon;
+use App\Models\Message;
+
+class MessagesTableSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        if (Message::all()->isEmpty()) {
+          $this->createMessages();
+        }
+    }
+
+    private function createMessages()
+    {
+      factory(Message::class)->create([
+        'title'     => 'Test',
+        'content'   => 'This is a test for MessagesTableSeeder',
+      ]);
+    }
```

Comment:
Je pense que tu peux renommer la fonction à `createMessage()` puisque l'on créé seulement 1 message dans la fonction
```suggestion
    private function createMessage()
    {
      factory(Message::class)->create([
        'title'     => 'Test',
        'content'   => 'This is a test for MessagesTableSeeder',
      ]);
    }
```